wallettool/src/main/java/org/bitcoinj/wallettool/WalletTool.java

```diff
@@ -1088,6 +1096,10 @@ private void shutdown() {
     }
 
     private void createWallet(NetworkParameters params, File walletFile) throws IOException {
+        KeyChainGroupStructure keychainStructure = (structure.equals(Structure.BIP32)) ?
+                KeyChainGroupStructure.DEFAULT :
```

Comment:
Shouldn't that be the explicit `KeyChainGroupStructure.BIP32`?

Reply:
Yes, it should. That didn't exist when I made this PR.

Reply:
Fixed and force-pushed.